### ISSUE TITLE
`obconf` missing GSettings

### ISSUE THREAD

**Issue by mainrs** [NixOS/nixpkgs#188232]
### Describe the bug
Trying to run `obconf` and interacting with the GUI in any meaningful matter results in this error:
```
(obconf:23699): GLib-GIO-ERROR **: 00:37:55.569: No GSettings schemas are installed on the system
fish: Job 1, 'obconf' terminated by signal SIGTRAP (Trace or breakpoint trap)
```

### Steps To Reproduce
1. `environment.systemPackages = [ pkgs.obconf ]`
2. Start `obconf` from the terminal
3. Either press `Install theme` or `Create theme archive`.

### Notify maintainers

@sfrijters

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.60, NixOS, 22.05 (Quokka), 22.05.20220821.23534df`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.10.3`
 - channels(me): `"nixos-21.05.4470.3b422991781, nixos-unstable-21.11pre323695.9aeeb7574fb"`
 - channels(root): `"nixos-21.11.334587.e6377ff3554, nixos-unstable-22.05pre338911.03ec468b140"`
 - nixpkgs: `/nix/store/mrf1rcmvfwsgdcrcvgg06mfz12im6inf-source`
```

**Comment by SFrijters** [comment]
@mainrs It seems I cannot add you as reviewer, but could you take a look at the fix in #188266?

**Comment by mainrs** [comment]
I sadly cannot test it right now. The earliest is start of October.
